src/reducers/configuration.ts

```diff
@@ -0,0 +1,23 @@
+import { handleActions } from 'redux-actions';
+import immutable from 'immutability-helper';
+
+import { ActionTypes } from '../constants';
+import { RepositoryConfigurationModel } from '../generated/tdr';
+
+export interface ConfigurationState {
+  configObject: RepositoryConfigurationModel;
+}
+
+export const initialConfigurationState: ConfigurationState = {
+  configObject: {},
```

Comment:
Because a) none of the other reducers do that b) I don't want the top level state to be driven by an autogenerated code and c) I want to reserve the ability to have other variables in here (e.g. loading)